Спринт 7. Консольное приложение для заметок. Версия 1.0.

andreysuzi opened the pull request.

andreysuzi committed:
Спринт 7. Консольное приложение для заметок. Версия 1.0.

AleksandrIlinskii commented on src/main/kotlin/Menu.kt:
```diff
@@ -0,0 +1,108 @@
+        println("Введите код действия:")
+        val code = takeCode()
+        when(code) {
+            0 -> editNote(note!!.text)
```
В функцию ты передаешь Note. Он точно не будет null, поэтому !! тут излишне

AleksandrIlinskii commented on src/main/kotlin/Menu.kt:
```diff
@@ -0,0 +1,108 @@
+                    println("Введите номер строки, перед которой хотите вставить новую строку:")
+                    val number = takeCode()
+                    if (number in 1 .. text.size) {
+                        println("Введите строку, которую хотите доавить:")
```
доавить - опечатка

andreysuzi committed:
Спринт 7. Консольное приложение для заметок. Версия 1.1.
Пользователь не может ввести пустое имя архива или заметки.
После создания новой заметки пользователь попадает в заметку, где должен ввести ее текст.
Добавлена возможность сразу вводить несколько строк в заметку или добавлять несколько строк в конец существующей заметки.
В меню добавлено уточнение "Создать архив" / "Создать заметку".
Исправлен формат кода.

AleksandrIlinskii commented on src/main/kotlin/Main.kt:
```diff
@@ -1,56 +1,63 @@
-                    println("Заметка пустая")
-                }
-                else {
+                if (currentNote!!.text.size == 0) {
```
У String есть свойство length, возвращающее длину строки
